### PR TITLE
LPS-97199 AB Testing DO NOT BACKPORT

### PR DESCRIPTION
**Pull request by darquesdev**

### CODE REVIEW

**Comment by darquesdev** [comment]
ci:test:relevant

**Comment by darquesdev** [comment]
ci:test:sf

File: modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/model/listener/SegmentsExperimentModelListener.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+		throws ModelListenerException {
+
+		try {
+			_asahSegmentsExperimentProcessor.processAddSegmentsExperiment(
+				segmentsExperiment);
+		}
+		catch (Exception e) {
+			if (_log.isWarnEnabled()) {
+				_log.warn(
+					"Unable to process addition of segments experiment" +
```
Comment (darquesdev) [inline modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/model/listener/SegmentsExperimentModelListener.java]: please, review the log message

**Comment by epgarcia** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by epgarcia** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/75942. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**